schematic/store/synapse.py

```diff
@@ -19,12 +19,10 @@
 import numpy as np
 import pandas as pd
 import synapseclient
-import synapseutils
```

Comment:
If you're using vscode, you may see unused imports "grayed" out.  It's ok to remove them in PRs, the intent is that each PR that we contribute should put the codebase in a better place.

Reply:
Pylint would also catch this, but this module hasn't been gone through and fixed so that is passes, so it isn't included in the pylint check.